lib/service/account_service.dart

```diff
@@ -150,20 +151,29 @@ class AccountServiceImpl extends AccountService {
   }
 
   @override
-  Future<Persona> importPersona(String words,
+  Future<Persona> importPersona(String words, String password,
       {WalletType walletType = WalletType.Autonomy}) async {
+    late String firstEthAddress;
+    try {
+      final uuid = const Uuid().v4();
```

Comment:
This will make confuse. Please move uuid to kotlin/swift code. Just call LibAukDart.calculateFirstEthAddress(words, passwork). In this method, generate an uuid, then call getWallet(uuid).calculateFirstEthAddress